BusBoard.ConsoleApp/App.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" />
```

Comment:
I think I would have committed these config changes in a separate commit, just to make your intentions clearer and your git history clearer